azure-pipelines.yml

```diff
@@ -3,6 +3,8 @@ variables:
   value: true
 - name: _TeamName
   value: AspNetCore
+- name: DOTNET_SKIP_FIRST_TIME_EXPERIENCE
```

Comment:
This the right way to add this environment variable for azure pipelines? Went over the docs and this seemed correct but would be good to have a second set of eyes.

Reply:
I believe that should do it. I've always seen this env passed 1, does it just check for existence?

Reply:
Agree. Please double-check that `true` works locally.

Reply:
BTW yes, this is the correct way to add an environment variable.

Reply:
Yup, they support `1`, `true` and `yes` 😆 ,

Where they check https://github.com/dotnet/cli/blob/4baa14f25f634155ef2e29d9ca400467e7a441ca/src/dotnet/Program.cs#L148 
The logic behind the environment variable extraction: https://github.com/dotnet/cli/blob/95c6eff6daa1a69f29c42b2d405400ad44bdec91/src/Microsoft.DotNet.Cli.Utils/EnvironmentProvider.cs#L132-L141